sdk/src/asset_handlers/bmff_io.rs

```diff
@@ -1066,6 +1066,135 @@ impl AssetIO for BmffIO {
         let vec: Vec<HashObjectPositions> = Vec::new();
         Ok(vec)
     }
+
+    fn remove_cai_store(&self, asset_path: &Path) -> Result<()> {
```

Comment:
This should really be a READ+SEEK trait but I realize there's temp_files inside so there's no point. I guess eventually we need trait abstraction for a temp file? Not to worry about now, but I want us to get there evenutally.